codespell_lib/data/dictionary.txt

```diff
@@ -20177,7 +20177,7 @@ draconain->draconian
 dragable->draggable
 draged->dragged
 draging->dragging
-draing->drawing
+draing->drawing, draining,
```

Comment:
Indeed, but in this example I think the intended word was `drain`, not `draining`.
```suggestion
draing->drawing, drain, draining,
```

Reply:
![Screenshot_20230902_104053](https://github.com/codespell-project/codespell/assets/4140247/a692fc6d-04de-4b68-ad44-cb11b51864ac)

Can't seem to commit via GH interface

Reply:
Added manually in adb7ec0

Reply:
Yes, the GitHub interface is broken, it cannot modify files > 1 MB.

Perhaps it's time to split dictionary.txt into multiple files.